sql/catalyst/src/main/scala/org/apache/spark/sql/internal/connector/PredicateUtils.scala

```diff
@@ -43,6 +43,73 @@ private[sql] object PredicateUtils {
           Some(In(attribute, Array.empty[Any]))
         }
 
+      case "=" | "<=>" | ">" | "<" | ">=" | "<=" if predicate.children().length == 2 &&
```

Comment:
Both `case "=" | "<=>" | ">" | "<" | ">=" | "<="` and `"STARTS_WITH" | "ENDS_WITH" | "CONTAINS"` use condition

```
if predicate.children().length == 2 &&
  predicate.children()(0).isInstanceOf[NamedReference] &&
  predicate.children()(1).isInstanceOf[LiteralValue[_]]
```
Is it necessary to extract a function?